static/styles/compose.css

```diff
@@ -378,10 +378,53 @@
             }
         }
     }
+
+    &.info {
+        background-color: hsl(204, 58%, 92%);
+        border-color: hsla(204, 49%, 29%, 0.4);
+        position: relative;
+        color: hsl(204, 49%, 29%);
+
+        .compose_banner_close_button {
+            color: hsla(204, 49%, 29%, 0.5);
+
+            &:hover {
+                color: hsl(204, 49%, 29%);
+            }
+
+            &:active {
+                color: hsla(204, 49%, 29%, 0.75);
+            }
+        }
+    }
+}
+
+.upload_banner {
+    overflow: hidden;
+
+    &.hidden {
+        display: none;
+    }
+
+    .moving_bar {
+        position: absolute;
+        width: 0;
+        /* The progress updates seem to come every second or so,
+        so this is the smoothest it can probably get. */
+        transition: width 1s ease-in-out;
```

Comment:
Looking at `node_modules/@uppy/progress-bar/src/style.scss`, it looks like they use a 0.4s animation duration in their own progress bar implementation; (and another for 0.2s on "height"; not sure what that does). How did you determine the 1s here? 

``` css
$ cat node_modules/@uppy/progress-bar/src/style.scss
@import '@uppy/core/src/_utils.scss';
@import '@uppy/core/src/_variables.scss';

.uppy-ProgressBar {
  /* no important */
  position: absolute;
  top: 0;
  left: 0;
  z-index: 10000;
  width: 100%;
  height: 3px;
  transition: height 0.2s;
}

.uppy-ProgressBar[aria-hidden=true] {
  /* no important */
  height: 0;
}

.uppy-ProgressBar-inner {
  width: 0;
  height: 100%;

  /* no important */
  background-color: $blue;
  box-shadow: 0 0 10px rgba($blue, 0.7);
  transition: width 0.4s ease;
}

.uppy-ProgressBar-percentage {
  position: absolute;
  top: 50%;
  left: 50%;

  /* no important */
  display: none;
  color: $white;
  text-align: center;
  transform: translate(-50%, -50%);
}
```

Reply:
```
 /* The progress updates seem to come every second or so,
        so this is the smoothest it can probably get. */
```

It's not perfect, but I wanted to get something close to a smooth moving bar, instead of jumping the bar every time a progress update comes in.

Reply:
OK; it seems to feel OK when watching it go both with fast and slow uploads, so probably this is fine.